extensions/peerjs.js

```diff
@@ -0,0 +1,963 @@
+// Name: PeerJS
+// ID: mikedevpeerjs
+// Description: A port of the PeerJS library for Scratch.
```

Comment:
Maybe instead just
```diff
- // Description: A port of the PeerJS library for Scratch.
+ // Description: Use PeerJS in your projects.
```

Because technically these extensions don't WORK on Scratch, more so TurboWarp and similar mods.